### PR TITLE
add r-aer  r-afex  r-highcharter  r-influencer  r-lintr  r-networkd3  r-plm  r-rbokeh

### PR DESCRIPTION
**Pull request by bgruening**
r-aer
 r-afex
 r-highcharter
 r-influencer
 r-lintr
 r-networkd3
 r-plm
 r-rbokeh

xref: https://github.com/conda-forge/staged-recipes/issues/2009
ping @johanneskoester 

Level 7!

### CODE REVIEW

File: recipes/r-afex/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - r-car
+    - r-coin
+    - r-lme4 >=1.1_8
+    - r-lmertest
```
Comment (isuruf) [inline recipes/r-afex/meta.yaml]: This package is not in conda-forge

**Comment by isuruf** [comment]
`r-lmertest` is missing in conda-forge

**Comment by bgruening** [comment]
The Windows builds are currently blocked by https://ci.appveyor.com/project/conda-forge/r-hmisc-feedstock - we need a build here.

**Comment by bgruening** [comment]
And an other blocker: https://travis-ci.org/conda-forge/r-igraph-feedstock.
Sorry I do not have permissions to restart.

**Comment by isuruf** [comment]
`r-rbokeh` depends on `r-maps`, but it is not available on windows

**Comment by bgruening** [comment]
@isuruf all tests passed now! :)

**Comment by bgruening** [comment]
Thanks a lot @isuruf!

**Pull request merged**